test/components/sortable-densities.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import SortableDensities from '../../js/components/sortable-densities'
+import config from '../../js/config'
+
+const originalConfig = Object.assign({}, config)
+const store = {
+  model: {
+    plates: []
+  }
+}
+
+describe('SortableDensities component', () => {
+  beforeEach(() => {
+    // Restore config as it can be modified by tests.
+    Object.assign(config, originalConfig)
```

Comment:
To fully restore the config, including new keys, would you want to do something like `Object.assign({}, originalConfig)` instead?

Reply:
Good point, thanks!

Reply:
That wouldn't work actually, as it's not enough to overwrite local `config` reference. I've added another approach to delete new keys. If we use this pattern more, I guess we should move it to some helper method.